README.md

```diff
@@ -86,6 +86,20 @@ the same aspect ratio in KaTeX as in LaTeX.  However, because most browsers
 default to a larger font size, this typically means that a 1cm kern in KaTeX
 will appear larger than 1cm in browser units.
 
+### Common Issues
+- Many Markdown preprocessors, such as the one that Jekyll and GitHub Pages use,
+  have a "smart quotes" feature.  This changes `'` to `’` which is an issue for
+  math containing primes, e.g. `f'`.  This can be worked around by defining a
+  single character macro which changes them back, e.g. `{"’", "'"}`.
+- KaTeX follows LaTeX's rendering of aligned and *matrix environments.  When
```

Comment:
Extra `*` before "matrix".  Should we add "unlike MathJax" here?

Reply:
I'm going to put `*matrix` inside backticks to get around the italics issues.

Reply:
What is `*matrix`? Maybe it's `matrix*`? (I'm just guessing based on LaTeX conventions.) Does this comment not apply to regular `matrix`?  Also I'd suggest adding backticks to aligned to match.

Reply:
Doesn't `*matrix` cover `matrix`, `bmatrix`, `pmatrix`, etc.?  I guess in this case `*` is overloaded... I can just write **all** `matrix` environments.

Reply:
I'm going to leave the **all** out.